src/main/g8/README.md

```diff
@@ -261,13 +271,4 @@ object Main extends LazyLogging {
 
 ## Who do I talk to? ##
 
-- Email: chap.s.b@gmail.com
-- Github: https://github.com/sbchapin/
-- Bitbucket: https://bitbucket.org/sbchapin/
-- Discord: sbchapin#7435
-
```

Comment:
I removed your specific info since I assumed you didn't want this to be in the generated result. Keyword: assumed

Reply:
Perfect!